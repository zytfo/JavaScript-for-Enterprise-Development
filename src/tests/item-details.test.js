import React from 'react';
import renderer from 'react-test-renderer';
import { Provider } from 'react-redux';
import createStore from 'redux-mock-store';
import thunk from 'redux-thunk';
import { BrowserRouter as Router, Route } from 'react-router-dom';

import ItemDetails from '../components/item-details'

describe('ItemDetails', () => {
    it('test render empty component', () => {
        const initialState = {
            item: void 0
        };

        const container = renderer
            .create(
                <Router>
                    <Route component={ItemDetails}/>
                </Router>
            );

        expect(container.toJSON()).toMatchSnapshot()
    })

});