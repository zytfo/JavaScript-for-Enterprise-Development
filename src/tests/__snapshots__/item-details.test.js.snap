// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ItemDetails test render empty component 1`] = `null`;
